<launch>
    <arg name="scene" default="bolter2_2" />
    <arg name="urdf_file" value="xacro --inorder '$(find scene_builder)/output/$(arg scene)/main_robot.xacro'" />

    <param name="robot_description" command="$(arg urdf_file)" />

    <node name="robot_state_publisher" pkg="robot_state_publisher" type="robot_state_publisher" />
    <!-- <node name="joint_state_publisher" pkg="joint_state_publisher" type="joint_state_publisher" /> -->
    <node name="joint_state_publisher_gui" pkg="joint_state_publisher_gui" type="joint_state_publisher_gui" />

    <node name="view_scene" pkg="rviz" type="rviz" args="-d $(find scene_builder)/rviz/view_scene.rviz" />
</launch>